tests/modules/misc.c

```diff
@@ -288,6 +288,31 @@ int test_log_tsctx(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     return REDISMODULE_OK;
 }
 
+int test_aligned_alloc(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+    REDISMODULE_NOT_USED(argv);
+    REDISMODULE_NOT_USED(argc);
+
+    size_t size = 100;
+
+    for(int i = 0; i < 6; i++) {
+        // Alignment, which must be a power of two and a multiple of sizeof(void *)
+        int alignment = sizeof(void*) * 2<< i;
```

Comment:
`*` priority is greater than `<<`, 
```suggestion
        int alignment = sizeof(void*) * (2 << i);
```